src/Core/tests/DeviceTests/Handlers/Button/ButtonHandlerTests.Android.cs

```diff
@@ -1,12 +1,40 @@
 using System.Threading.Tasks;
 using AndroidX.AppCompat.Widget;
+using Microsoft.Maui.DeviceTests.Stubs;
 using Microsoft.Maui.Handlers;
+using Xunit;
 using AColor = global::Android.Graphics.Color;
 
 namespace Microsoft.Maui.DeviceTests
 {
 	public partial class ButtonHandlerTests
 	{
+		[Fact(DisplayName = "Button Padding Initializing")]
+		public async Task PaddingInitializesCorrectly()
```

Comment:
Is it possible to make this a shared test? The implementations of `GetNativePadding` could convert to the uniform units in a tuple or even just use the `Thickness` type.